integration-test/1194-bus-route-refs.py

```diff
@@ -46,10 +126,38 @@ def test_full_lists_disappear_by_zoom_12(self):
         # note that it doesn't matter what the bus shield is - that's
         # data-dependent. for the purposes of the test, we only care that
         # there _is_ one.
-        self.load_fixtures(['http://www.openstreetmap.org/way/225516711',
-                            'http://www.openstreetmap.org/relation/2980505',
-                            'http://www.openstreetmap.org/relation/2980504'])
-
+        self.generate_fixtures(
+            dsl.way(225516711, wkt_loads('LineString (-122.44117830000000424 37.79163929999999993, -122.44752560000000585 37.79082819999999998)'), {
```

Comment:
Consider a `tile_diagnal` for this feature as well (rather than searching for right parent tile).